netweaver/ha_cluster.sls

```diff
@@ -7,6 +7,17 @@
 {% set instance_name = node.sid~'_'~instance %}
 {% set instance_folder = node.sap_instance.upper()~instance %}
 {% set profile_file = '/usr/sap/'~node.sid.upper()~'/SYS/profile/'~node.sid.upper()~'_'~instance_folder~'_'~node.virtual_host %}
+{% set virtual_host_interface = node.virtual_host_interface|default('eth0') %}
+{% set ifcfg_file = '/etc/sysconfig/network/ifcfg-'+virtual_host_interface %}
```

Comment:
probably better to use the `~` instead of `+`

Reply:
Yes, totally right! Good catch